src/pages/SearchPage.vue

```diff
@@ -119,7 +119,7 @@ const computedEntries = computed(() => {
   padding-bottom: 0;
   padding-top: 0;
   @media (max-width: 500px) {
-    padding: 0;
+    padding: 1;
```

Comment:
I guess you forgot to specify the exact value which breaks the UI for search page. But I'd rather suggest leave it at 0 as changing this value to something else changes the search page padding entirely.